src/content/guides/build-performance.md

```diff
@@ -152,6 +155,57 @@ new CommonsChunkPlugin({
 })
 ```
 
+### Avoid Extra Optimization Steps
+
+webpack does extra algorithmic work to optimize the output for size and load performance. These are usually not needed during iterative development:
+
+```js
+optimization: {
```

Comment:
Can you add the outer brackets here like you did in other examples?

Reply:
Sure!